en/core-libraries/time.rst

```diff
@@ -140,6 +140,23 @@ dates. CakePHP makes this a snap::
     // Outputs '2014-04-20 22:10'
     $now->i18nFormat('YYYY-MM-dd HH:mm:ss');
 
+As of 3.1.0 you can format dates with non-gregorian calendars::
+
+    // Outputs 'Thursday, Dey 24, 1388 at 1:59:28 PM GMT'
```

Comment:
having another look into this, i don't get the same date.
This would match to 2010-01-14 for me but anyway I believe that most of the above example is wrong
``$now = Time::parse('2014-10-31');$now->i18nFormat();`` wont output ``4/20/14, 10:10 PM``.

I can check and update this tonight if you want.

![image](https://cloud.githubusercontent.com/assets/4977112/8953699/c9f8839e-35e0-11e5-9052-41ac0c11daaf.png)